trust-store-trust-policy-specification.md

```diff
@@ -11,14 +11,16 @@ The document consists of the following sections:
 Users who consume and execute the signed artifact from a registry need a mechanism to specify the trusted producers.
 This is where Trust Store is used.
 
-Trust store allows users to specify two kinds of identities:
+Trust store allows users to specify two kinds of identities, with additional identities coming in future releases:
 
 - **Certificates**: These are the signing certificates or certificates in the certificate chain of the signing certificate.
```

Comment:
These are typically root certificates of of issuing authorities (public CA, internal CAs) that the user trusts, and which will issue the signing certificates.